fundingmanager_test.go

```diff
@@ -18,12 +18,10 @@ import (
 	"github.com/btcsuite/btcd/chaincfg"
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
 	"github.com/btcsuite/btcd/wire"
-	"github.com/btcsuite/btclog"
 	"github.com/btcsuite/btcutil"
-	_ "github.com/btcsuite/btcwallet/walletdb/bdb"
```

Comment:
Reason for this change?

Reply:
dunno, go1.11 probably removed it. doesn't seem critical as tests are passing? :)

Reply:
Yep, probably a leftover import that wasn't removed earlier because of the blank :)